src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -269,23 +296,21 @@ private BibtexEntry cleanup(BibtexEntry entry) {
         }
 
         // clean up author
-        /*   	String author = (String)entry.getField("author");
-           	if (author != null) {
-            if (author.indexOf("a href=") >= 0) {  // Author parsing failed because it was empty
-        	entry.setField("author","");  // Maybe not needed anymore due to another change
-            } else {
-            	author = author.replaceAll("\\s+", " ");
-            	author = author.replaceAll("\\.", ". ");
-            	author = author.replaceAll("([^;]+),([^;]+),([^;]+)","$1,$3,$2"); // Change order in case of Jr. etc
-            	author = author.replaceAll("  ", " ");
-            	author = author.replaceAll("\\. -", ".-");
-                       author = author.replaceAll("; ", " and ");
-            	author = author.replaceAll(" ,", ",");
-            	author = author.replaceAll("  ", " ");
-            	author = author.replaceAll("[ ,;]+$", "");
-            	entry.setField("author", author);
-            }
-        }*/
+        String author = entry.getField("author");
+        if (author != null) {
+            author = author.replaceAll("\\s+", " ");
+            author = author.replaceAll("\\.", ". ");
+            //author = author.replaceAll("([^;]+),([^;]+),([^;]+)", "$1,$3,$2"); // Change order in case of Jr. etc
```

Comment:
It was giving strange results I thought it might not be required any longer. But today I dug deep for some records with Jr. Sr. in the name and confirmed it is still an issue.  I just added new code for proper parsing for this Jr. Sr. re-ordering and managed to find a few samples to confirm the new code is working.